utils/bash-completion/files/uci

```diff
@@ -0,0 +1,77 @@
+_uci_command_complete() {
```

Comment:
This file should be upstreamed. We don't need to ship our own file if others upstreamed their work, right? :)
By upstreaming this file, you would get plenty of advantages of upstream developers like treewide refactors, etc. Yeah, you can admit that having this file is useless upstream, but I could say the same with other completions, which are already there. :)